docs/website/layouts/partials/docs/article.html

```diff
@@ -16,6 +16,9 @@
       <div class="content">
         {{ .Content }}
       </div>
+      {{ if (and (not .Params.hide_feedback) (site.Params.ui.feedback.enable) (site.GoogleAnalytics)) }}
+        {{ partial "feedback.html" site.Params.ui.feedback }}
+      {{ end }}
```

Comment:
Some pages have more whitespace between the contents of the article and the Feedback section. We can move the Feedback section right under the Content of a document to fix this, but this adds a Feedback section to the table of contents on the side navbar.

Reply:
I've just noticed that from the preview. I'd like to understand why the content has such a giant padding-bottom...seems like something we could just drop? 🤔 will dive into our stylesheets a bit tomorrow.

Reply:
Seems like whitespace is being dynamically added so the scrolling can reach the last heading for the table of contents to work properly. I added another div that makes this more explicit and also fixes the whitespace between the article and feedback.

Reply:
💡 I had no idea. Thanks for digging into this and fixing it, too. 👍